instagram/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import dummyData from "./dummy-data";
+import "./index.css";
+import Header from "./components/header";
+import PostContainer from "./components/postContainer";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      users: dummyData
+    };
+  }
+
+  addNewComment = () => {
+    const newComment = {
```

Comment:
I think you need to pass in newComment as argument to this function. Then add this newComment to the post that it belongs to.